spec/griddler/email_spec.rb

```diff
@@ -484,6 +484,15 @@ def email_with_params(params)
 
     expect(headers[header_name]).to eq({ "a" => ["invalid utf-8 bytes are ÀÁõúþÿ."] })
   end
+  
+  it 'deeply cleans invalid UTF-8 bytes from an array when it is submitted' do
+    header_name = 'Arbitrary-Header'
+    header_value = "invalid utf-8 bytes are \xc0\xc1\xf5\xfa\xfe\xff."
+    header = [{ header_name => { "a" => [header_value] } }]
+    headers = header_from_email(header)
+
+    expect(headers[0][header_name]).to eq({ "a" => ["invalid utf-8 bytes are ÀÁõúþÿ."] })
```

Comment:
Line is too long. [89/80]<br>Redundant curly braces around a hash parameter.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.